src/io/flutter/run/test/TestLaunchState.java

```diff
@@ -113,15 +113,15 @@ protected ConsoleView createConsole(@NotNull Executor executor) throws Execution
     final DartUrlResolver resolver = DartUrlResolver.getInstance(project, testFileOrDir);
     final ConsoleProps props = ConsoleProps.forPub(config, executor, resolver);
     final BaseTestsOutputConsoleView console = SMTestRunnerConnectionUtil.createConsole(ConsoleProps.pubFrameworkName, props);
+
     final Module module = ModuleUtil.findModuleForFile(testFileOrDir, project);
-    if (module != null) {
-      console.addMessageFilter(new FlutterConsoleFilter(module));
```

Comment:
Are you sure you want to remove this? It has some specialized logic for handling Flutter stuff, like plugins.

Reply:
I'll figure out what all the FlutterConsoleFilter is responsible for. I _think_ it's more important for the run console (to link to lib.main.dart, and some flutter doctor output?). It may have several different concerns in that class.

Reply:
I added docs to FlutterConsoleFilter. I _think_ we only want to use it for the run and debug consoles.